contracts/governance/GovernorAlpha.sol

```diff
@@ -0,0 +1,448 @@
+        bytes[] calldatas;
+
+        // @notice The block at which voting begins: holders must delegate their votes prior to this block
+        uint startBlock;
+
+        // @notice The block at which voting ends: votes must be cast prior to this block
+        uint endBlock;
+
+        // @notice Current number of votes in favor of this proposal
+        uint forVotes;
+
+        // @notice Current number of votes in opposition to this proposal
+        uint againstVotes;
+
+        // @notice Flag marking whether the proposal has been canceled
+        bool canceled;
+
+        // @notice Flag marking whether the proposal has been executed
+        bool executed;
+
+        // @notice Receipts of ballots for the entire set of voters
+        mapping (address => Receipt) receipts;
+    }
+
+    // @notice Ballot receipt record for a voter
+    struct Receipt {
+        // @notice Whether or not a vote has been cast
+        bool hasVoted;
+
+        // @notice Whether or not the voter supports the proposal
+        bool support;
+
+        // @notice The number of votes the voter had, which were cast
+        uint96 votes;
+    }
+
+    // @notice Possible states that a proposal may be in
+    enum ProposalState {
+        Pending,
+        Active,
+        Canceled,
+        Defeated,
+        Succeeded,
+        Queued,
+        Expired,
+        Executed
+    }
+
+    // @notice The official record of all proposals ever proposed
+    mapping (uint => Proposal) public proposals;
+
+    // @notice The latest proposal for each proposer
+    mapping (address => uint) public latestProposalIds;
+
+    // @notice The EIP-712 typehash for the contract's domain
+    bytes32 public constant DOMAIN_TYPEHASH = keccak256("EIP712Domain(string name,uint256 chainId,address verifyingContract)");
+
+    // @notice The EIP-712 typehash for the ballot struct used by the contract
+    bytes32 public constant BALLOT_TYPEHASH = keccak256("Ballot(uint256 proposalId,bool support)");
+
+    // @notice An event emitted when a new proposal is created
+    event ProposalCreated(uint id, address proposer, address[] targets, uint[] values, string[] signatures, bytes[] calldatas, uint startBlock, uint endBlock, string description);
+
+    // @notice An event emitted when a vote has been cast on a proposal
+    event VoteCast(address voter, uint proposalId, bool support, uint votes);
+
+    // @notice An event emitted when a proposal has been canceled
+    event ProposalCanceled(uint id);
+
+    // @notice An event emitted when a proposal has been queued in the Timelock
+    event ProposalQueued(uint id, uint eta);
+
+    // @notice An event emitted when a proposal has been executed in the Timelock
+    event ProposalExecuted(uint id);
+
+    /**
+     * @dev Initialize sets the addresses of timelock contract, trusttoken contract, and guardian
+     */
+    function initialize(address timelock_, address trustToken_, address guardian_, uint _votingPeriod) external {
```

Comment:
Any reason why do we have `timelock_`, `trustToken_`, `guardian_`, but `_votingPeriod`?
(`_` placing)